sdk/test/opentelemetry/sdk/trace/samplers_test.rb

```diff
@@ -83,35 +77,30 @@
       result.wont_be :sampled?
     end
 
-    it 'does not sample a root span unless apply_to_root_spans' do
```

Comment:
This is not a supported case for `ProbabilitySampler`. The probability applies to "root spans", "root spans and remote parent", or "all spans".